client/components/domains/map-domain-step/index.jsx

```diff
@@ -132,6 +133,19 @@ class MapDomainStep extends React.Component {
 					</div>
 
 					{ this.domainRegistrationUpsell() }
+
+					<div className="map-domain-step__domain-text">
+						{ translate(
+							'Domain registration and billing will remain at your current provider,' +
+								" and you'll update your domain's DNS to point it to your WordPress.com site. " +
```

Comment:
DNS or Nameservers? :) I think we say `nameservers` everywhere else.

Reply:
Good question! On the previous page at `/start/domains/use-your-domain` we say "Requires changes to the domain's DNS" so that's where I nicked that from.

Reply:
Nameservers are part of DNS, but in this case, I'd totally go with nameservers here. @Automattic/cobalt what do you think?

Reply:
We also mention DNS in the first paragraph of the support doc...should all of those change, do you think? "Nameservers" might be less jargon-y/more familiar.

Reply:
I wouldn't do much with the support docs. But in this specific place, it's better to say nameserves, and provide the support link.

Reply:
Would it be better to switch up the phrases?

ie - "Update your domain's DNS (or name servers, whatever we decide to  use here) to point it to your WordPress.com site. Domain registration and billing will remain at your current provider. Learn more"